### PR TITLE
rework package: now using virtual KDL chain and FJT action client

### PR DESCRIPTION
**Pull request by fmessmer**
this package now provides an action interface that allows to point a frame `pointing_frame` on the robot towards a given `target_frame` executing the solution via `FollowJointTrajectory` on e.g. torso or head (**not** combined)

### CODE REVIEW

File: cob_lookat_action/CMakeLists.txt [hunk 1]
```diff
@@ -1,7 +1,19 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(cob_lookat_action)
 
-find_package(catkin REQUIRED COMPONENTS actionlib_msgs actionlib control_msgs geometry_msgs message_generation moveit_msgs roscpp tf trajectory_msgs)
+find_package(catkin REQUIRED COMPONENTS
+  actionlib_msgs
```
Comment (mathias-luedtke) [inline cob_lookat_action/CMakeLists.txt:5]: not sure if this is properly sorted

File: cob_lookat_action/CMakeLists.txt [hunk 2]
```diff
@@ -15,22 +27,32 @@ generate_messages(
...
 )
 
-install(PROGRAMS src/cob_lookat_action_client.py src/cob_monitor_arm_client.py
-  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}/src
+install(DIRECTORY include/${PROJECT_NAME}/
+  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
+)
+
+install(PROGRAMS scripts/cob_lookat_action_client.py scripts/cob_lookat_target_publisher.py
+  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}/scripts
```
Comment (mathias-luedtke) [inline cob_lookat_action/CMakeLists.txt]: I should go into `CATKIN_PACKAGE_BIN_DESTINATION ` directly.
If you code is ptyhon3 compatible, you can try http://docs.ros.org/indigo/api/catkin/html/howto/format2/installing_python.html

**Review by mathias-luedtke — APPROVED** [review]
Overall okay.
So you want me to review the code?

**Comment by fmessmer** [comment]
fixes https://github.com/ipa320/cob_manipulation/issues/71

**Pull request merged**